.github/workflows/check-issues-nightly.yml

```diff
@@ -0,0 +1,23 @@
+name: Check issues with pending validation labels
+
+on:
```

Comment:
```suggestion
on:
  workflow_dispatch: {}
```

have you tried to execute this ci job? if not, maybe it's worth adding custom dispatch and trying it once.

Reply:
Yeah I executed this locally by using [act](https://github.com/nektos/act)

Reply:
It's still worth adding the `workflow_dispatch` option so we can run it with `et workflow-dispatch` command 😉

Reply:
Sure, added